functional-tests/fn_bridge_deposit_happy.py

```diff
@@ -44,114 +46,97 @@ def main(self, ctx: flexitest.RunContext):
+        btc_user = btc.props["rpc_user"]
+        btc_password = btc.props["rpc_password"]
+        seq_addr = seq.get_prop("address")
 
-        # Create the first deposit request transaction
-        tx = deposit_request_transaction(el_address_2, btc_url, btc_user, btc_password)
+        # Create the deposit request transaction
+        tx = bytes(
+            deposit_request_transaction(
+                el_address, musig_bridge_pk, btc_url, btc_user, btc_password
+            )
+        ).hex()
         self.logger.debug(f"Deposit request tx: {tx}")
-        txid = btcrpc.sendrawtransaction(tx)
-        self.logger.debug(f"sent deposit request with txid = {txid} for address {el_address_2}")
-
-        # Now poll for the tx in chain
-        tx_published = False
-        for _ in range(10):
-            time.sleep(1)
-            try:
-                _ = btcrpc.gettransaction(txid)
-                print("Found expected tx in mempool")
-                tx_published = True
-                break
-            except Exception as e:
-                print(e)
-        assert tx_published, "Tx was not published"
-        # time to mature FIXME: is this 6 blocks?
-        btcrpc.proxy.generatetoaddress(6, seq_addr)
+
+        # Send the transaction to the Bitcoin network
+        txid = btcrpc.proxy.sendrawtransaction(tx)
+        self.logger.debug(f"sent deposit request with txid = {txid} for address {el_address}")
+        # this transaction is not in the bitcoind wallet, so we cannot use gettransaction
         time.sleep(1)
 
-        new_balance_2 = int(rethrpc.eth_getBalance(f"0x{el_address_2}"), 16)
-        assert new_balance_2 > original_balance_2, "balance did not increase"
+        # time to mature DRT
+        btcrpc.proxy.generatetoaddress(6, seq_addr)
+        time.sleep(10)
 
-        # Third deposit, address 1
-        # Create the first deposit request transaction
-        tx = deposit_request_transaction(el_address_1, btc_url, btc_user, btc_password)
-        self.logger.debug(f"Deposit request tx: {tx}")
-        txid = btcrpc.sendrawtransaction(tx)
-        original_balance_3 = int(rethrpc.eth_getBalance(f"0x{el_address_1}"), 16)
-        self.logger.debug(f"Balance before deposit: {original_balance_3}")
-        txid = btcrpc.sendrawtransaction(tx)
-        self.logger.debug(f"sent deposit request with txid = {txid} for address {el_address_1}")
-
-        # Now poll for the tx in chain
-        tx_published = False
-        for _ in range(10):
-            time.sleep(1)
-            try:
-                _ = btcrpc.gettransaction(txid)
-                print("Found expected tx in mempool")
-                tx_published = True
-                break
-            except Exception as e:
-                print(e)
-        assert tx_published, "Tx was not published"
-        # time to mature FIXME: is this 6 blocks?
+        # time to mature DT
         btcrpc.proxy.generatetoaddress(6, seq_addr)
+        time.sleep(10)
```

Comment:
This is not working and I don't know why. The sequencer does not pick up the transaction, despite being successfully broadcasted.
Any ideas @Rajil1213, @delbonis?

Reply:
Then it explodes here in L68
https://github.com/alpenlabs/strata/blob/d81b827fb59ad7a97d447dca208c0a11671fe3dd/functional-tests/fn_bridge_deposit_happy.py#L66-L68

Reply:
@Rajil1213 figured it out. The magic bytes change and I hard-coded the damn thing to be  `b"alpenstrata"` 🤦🏻‍♂️